packages/core/src/components/text-field/text-field.tsx

```diff
@@ -130,6 +130,15 @@ export class TdsTextField {
     this.tdsBlur.emit(event);
   }
 
+  /** Method to handle focus */
+  @Method()
+  async focusTextField() {
+    if (this.textInput) {
+      const event = new FocusEvent('focus');
+      this.textInput.dispatchEvent(event);
```

Comment:
To not mix tegel and browser events, maybe this should follow the convention of `tds{event}` -> `tdsFocus`?

Reply:
Could you show me how to do that while ensuring the feature still works?

Reply:
I would suggest `tdsFocus`, so the users listen for an event called: `tdsFocus`

In react:

```jsx
<TdsTextField onTdsFocus={() => {}}/>
```

in plain js
```javascript
document.querySelector('tds-text-field').addEventListener(('tdsFocus') => {})
```

In angular (not quite sure about how this is done in angular):
```html
<tds-text-field (tdsFocus)={() => {}}/>
```

Reply:
When focusTextField() is called, it triggers the handleFocus(event) method, which then emits the tdsFocus event. Isn't that the intended behavior you were looking for, @mJarsater?

Reply:
Yeah that's how I would imagine it

Reply:
Great! What adjustments are needed for this to be approved? I noticed I can’t rename the method focusTextField to tdsFocus because there’s already an attribute with that name. Any suggestions, @mJarsater?

Reply:
I’ve tried this, @mJarsater, but it doesn’t work anymore after the change.
```
const event = new CustomEvent('tdsFocus');
this.textInput.dispatchEvent(event);
```

Reply:
That worked! Done.